package.json

```diff
@@ -30,7 +30,8 @@
     "build": "jsreport-studio-build",
     "prepublishOnly": "jsreport-studio-build",
     "start": "jsreport-studio-start --httpPort 4000",
-    "test": "mocha test --timeout 20000 && standard"
+    "test": "mocha test --timeout 20000 && standard",
+    "capmo-publish": "rimraf dist/* && npm pack --pack-destination ./dist"
```

Comment:
**question:** can you elaborate on why you opted to pack the resulting code into a dist folder that is checked in? what's the benefit vs. the previous way of installing?

Reply:
@sebastian-schlecht The Docker image for jsreport has been changed to `jsreport/jsreport:3.8.0-full` in the [report-service-v2](https://github.com/capmo/report-service-v2/pull/27/files#diff-4c0a9ccb793748d76e722dc2a66dabaaaf2e9ee2bd9e39d6b7e622f8f56baa9a) I tried to just use the same approach as it made sense with `RUN npm i github:capmo/jsreport-docx#master`. But in the new image, this didn't work. The image was missing some `ssh` required stuff to make the npm install from github work. I tried to resolve it, but I failed. After that effort didn't work, I had to find alternative. This alternative felt easiest. We publish the tarball and npm installs straight from the tarbal, thereby bypassing the ssh issue.

I am all up for making this a super clean package publish, as a custom `@capmo/jsreport-docx` npm package. But I was not sure if that would "just work" as a jsreport plugin. I might have had to play with https://jsreport.net/learn/custom-extension to make it work, and all of that felt too much effort on a thing I already spent way too much time. So I figured that can be left for the future.

What do you think?